src/balena-model.ts

```diff
@@ -393,6 +393,13 @@ export interface Release {
 	contract: {} | null;
 	is_passing_tests: boolean;
 	release_type: 'final' | 'draft';
+	is_finalized_at__date: DateString | null;
+	semver_major: number;
```

Comment:
Seems worthwhile to track these new fields in the `release history` as well. Keeping track of such metadata enables us to run analysis on these dimensions. Furthermore, we might want to expose `release history` to end users and they would appreciate such metadata.

Reply:
That would be in the cloud API PR @gelbal :)
You mean all the semver & the revision, or do you also care about the is_finalized_at__date ?

Reply:
Yeah, yeah for sure in the cloud API PR. Just wanted to share my thoughts as they appeared. Yes, I was thinking of tracking all the semver & the revision fields. Probably no need for `is_finalized_at__date` as we could derive it from the historical records. Besides, right now I cannot think of analytics reasons to keep track of that.